cypress/integration/headerNavigation.spec.js

```diff
@@ -0,0 +1,66 @@
+describe('Header Navigation', () => {
+  beforeEach(() => {
+    cy.viewport(1920, 1080)
```

Comment:
Interesting, didn't know that you could set the viewport during a test - that's a nice touch!